src/api-reference/budget/budget-header.md

```diff
@@ -711,163 +712,176 @@ concur-correlationid: 86a0d9fe-9e98-43c3-89d8-a2917dd844cb
 
 ### <a name="budgetItemHeaderList"></a>PagedBudgetItemHeaderList
 
-Name | Type | Format | Description
------|------|--------|------------
-`totalRows`	|	`integer`	|	-	|	The total number of rows available
-`offset`	|	`integer`	|	-	|	The starting row for this page of results (zero-based)
-`limit`	|	`integer`	|	-	|	The number of results returned per page.  (maximum 50)
-`budgetItemHeaders`	|	`Array[BudgetItemHeader]`	|	-	|	List of budget items 
-`href`	|	`string`	|	-	|	The href for this request
-`previous`	|	`integer`	|	-	|	The href for the previous page of results (null if this is the first page of results)
-`next`	|	`integer`	|	-	|	The href for the next page of results (null if no results remaining)
+Name|Type|Format|Description
+---|---|---|---
+`totalRows`|`integer`| - |The total number of rows available
```

Comment:
One last fix and then I think we will be good to go as far as I am concerned: remove this extraneous whitespace where the dashes are.

```
Current:   `totalRows`|`integer`| - |The total number of rows available
Should be: `totalRows`|`integer`|-|The total number of rows available
```

Reply:
:+1: